oak_docker_linux_init/README.md

```diff
@@ -7,46 +7,174 @@
 # Running a Docker container from stage0
 
 This example shows how to run a Docker container from Oak's stage0 firmware
-binary. We can achieve this by simply extracting the filesystem of the Docker
-container and packaging it as the
+binary. We can achieve this by preparing an
 [initial RAM disk](https://en.wikipedia.org/wiki/Initial_ramdisk) (initramfs)
-disk with an appropriate init binary. The easiest way to execute the initial RAM
-disk is to use
+that sets up the environment and launches the entry point of the docker image.
+Once we have the appropriate initial RAM disk, the easiest way to execute it is
+to use
 [QEMU's direct Linux boot feature](https://qemu-project.gitlab.io/qemu/system/linuxboot.html).
 
-Note: these steps assume that the commands will be executed from the project
+**Note:** these steps assume that the commands will be executed from the project
 root.
 
+## Prepare the initial RAM disk (initramfs)
+
+To run a Docker image on top of stage0 firmware binary, the key idea is to
+extract the filesystem from the Docker image and package it either as an
+initramfs image or a chroot tree.
+
+### Docker filesystem as the initial RAM disk
```

Comment:
We should probably keep investing in only one of these approaches. Probably the one below, since I expect we will need drivers and access to devices from the host (e.g. for the GPU use case). WDYT?

Reply:
Yes, the second approach of having a custom Docker launcher as initramfs and the Docker filesystem as a chroot tree is the viable long term approach. 

Just leaving this approach in for the time being as it can help with experimentation.